### ISSUE TITLE
Repo Switcher, Keep Existing Route

### ISSUE THREAD

**Issue by ashwell** [ZenHubIO/support#861]
So this might be a weird one but I think it is a good "nice to have". When using the repo switcher is always goes to the main repo page. I would love it if it would switch to the same page in the other repo.

An example with two repos "client" and "server"

1. I'm looking at the issues page for the "client" repo 
  * current route: **github.com/org/client/issues**

2. Open the repo switcher dropdown
    a. select "server" from the repo list

3. Route to **github.com/org/server/issues**
  * instead of the main page for that repo (github.com/org/server)

simply put, I would love to switch repos and keep the current github tab open (code, issues, pull requests, wiki, etc.)

If this would be a controversial feature I could see it being an opt-in configuration.

Only been using ZenHub for a week and I'm totally loving it! Thanks for all the hard work!

**Comment by azenMatt** [comment]
Thanks @ashwell :) 

I agree this could be very useful. 

We used to have it so it would always redirect to the board - but the feedback was going to the 'home' page was a better option. 

Lets put this one in the discussion pipeline for a while and see if we can get some 👍  or 👎  for how this would work out for everyone!

**Comment by adamreisnz** [comment]
+1 for this... it's quite annoying if I want to switch to a board of another that the repo switcher doesn't automatically take me there (provided I'm on the board of another repo).
